go/service/ui_router.go

```diff
@@ -155,3 +155,20 @@ func (u *UIRouter) GetRekeyUI() (keybase1.RekeyUIInterface, int, error) {
 	}
 	return ret, sessionID, nil
 }
+
+func (u *UIRouter) GetRekeyUINoSessionID() (keybase1.RekeyUIInterface, error) {
```

Comment:
Right, that's why I did it this way since the user is initiating these calls via CLI.

The gregor initiated calls, which result in a refresh loop, ask electron for a session id so that electron can bookkeep the window.